### DISCUSSION TITLE
Support view() / blade() test helpers

### DISCUSSION THREAD

**Discussion by gbalduzzi** [orchestral/testbench#413] [Ideas]
I'm building a Package for publishing custom blade components and I'm writing some tests.

However, this great package currently doesn't support [the utils for testing views](https://laravel.com/docs/11.x/http-tests#testing-views) such as `view()` and `blade()`.

It's my understanding that the testbench TestCase is missing the [`InteractsWithViews`](https://github.com/laravel/framework/blob/11.x/src/Illuminate/Foundation/Testing/Concerns/InteractsWithViews.php) concern.

I see [that some other packages](https://github.com/blade-ui-kit/blade-ui-kit/blob/main/tests/InteractsWithViews.php) embed the concern directly in their package.

I can submit a PR adding the `InteractsWithViews` concern to this library directly: let me know if there is interest or, on the contrary, there is some incompatibility that I don't notice

**Comment by crynobone** [comment]
Feel free to submit a PR to the Testbench Core repository. 

However, this has already been documented in our documentation: https://packages.tools/testbench/the-basic.html#testing-blade-components
